docs/tutorial/vscode.md

```diff
@@ -39,18 +39,18 @@ You can create a `.vscode/c_cpp_properties.json` file with `C/C++: Edit Configur
 {
     "configurations": [
         {
-            "name": "Linux",
+            "name": "N64 mm-us",
```

Comment:
The `mm` part feels kinda redundant given that mm is the only game that we will ever have on this repo
```suggestion
            "name": "n64 us",
```

Reply:
you can have local configurations that you can pick from within vscode, since the .vscode folder isnt committed. it was dragorns reason for renaming it in the first place: https://github.com/zeldaret/oot/pull/1669#discussion_r1472072110

so including the game name does make sense

Reply:
Yeah, but will ever have MM related configurations on your local `.vscode`. It is not like you would put an OoT configuration on `mm/.vscode`, so I don't understand the idea of putting the game name

Reply:
Since this is a recommendation anyways, I feel like it doesn't really matter too much. With that in mind I just decided to go with `n64-us`.